persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -177,8 +179,21 @@ selectUsersWithoutMetadata:
 SELECT * FROM User AS user
 WHERE deleted = 0 AND incomplete_metadata = 1;
 
+selectUsersWithOneOnOne:
+SELECT * FROM User
+WHERE deleted = 0 AND qualified_id != (SELECT id FROM SelfUser) AND qualified_id IN
+(
+	SELECT user FROM Member
```

Comment:
suggestion: I think since we don't need full details of the conversation, maybe joining with the conversation table would be better.